app/views/teachers/CreateTeacherAccountView.coffee

```diff
@@ -95,35 +113,41 @@ module.exports = class CreateTeacherAccountView extends RootView
 
   onSubmitForm: (e) ->
     e.preventDefault()
-    
+
     # Creating Trial Request first, validate user attributes but do not use them
     form = @$('form')
     allAttrs = forms.formToObject(form)
     trialRequestAttrs = _.omit(allAttrs, 'name', 'password1', 'password2')
-    
+
+    # Don't save n/a district entries, but do validate required district client-side
+    trialRequestAttrs = _.omit(trialRequestAttrs, 'district') if trialRequestAttrs.district?.replace(/\s/ig, '').match(/n\/a/ig)
+
     if @$('#other-education-level-checkbox').is(':checked')
       val = @$('#other-education-level-input').val()
       trialRequestAttrs.educationLevel.push(val) if val
-      
+
     forms.clearFormAlerts(form)
-    
+
     result = tv4.validateMultiple(trialRequestAttrs, formSchema)
     error = false
     if not result.valid
       forms.applyErrorsToForm(form, result.errors)
       error = true
-    if not forms.validateEmail(trialRequestAttrs.email)
-      forms.setErrorToProperty(form, 'email', 'Invalid email.')
+    if not error and not forms.validateEmail(trialRequestAttrs.email)
+      forms.setErrorToProperty(form, 'email', 'invalid email')
```

Comment:
Should errors in general be lower case, incomplete sentences? I was thinking as a general rule, error messages should be complete sentences, since some errors might need to be more than a sentence, and it looks a bit more cleanly. But it hasn't been decided on, as far as I know.